dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/helpers/calculate-initial-utilisation-and-fixed-fee.test.ts

```diff
@@ -1,9 +1,15 @@
-import { calculateInitialUtilisation } from '@ukef/dtfs2-common';
+import * as dtfsCommon from '@ukef/dtfs2-common';
 import { calculateInitialUtilisationAndFixedFee, parseDate, hasRequiredValues, RequiredParams } from './calculate-initial-utilisation-and-fixed-fee';
 import { TfmFacilitiesRepo } from '../../../../../repositories/tfm-facilities-repo';
 import { aTfmFacility } from '../../../../../../test-helpers';
-import { calculateInitialFixedFee } from './calculate-initial-fixed-fee';
-import { calculateUkefShareOfUtilisation } from '../../../../../helpers';
+import * as fixedFeeHelpers from './calculate-initial-fixed-fee';
+
+jest.mock('./calculate-initial-fixed-fee');
+// eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
Is this something that we just think will happen every time we want to mock something?
@abhi-markan if this is an expected piece of code for mocking in tests would you prefer we disable this rule across all test.ts files or leave this disable in?

Reply:
I copied this from another file but have since seen some other files are doing this:
```
jest.mock('@ukef/dtfs2-common', () => ({
  ...jest.requireActual<object>('@ukef/dtfs2-common'),
  calculateDrawnAmount: jest.fn(),
}));
```
I don't really know enough about this eslint rule to know if that is preferable or not

Reply:
Ah got you, okay i'm going to resolve and approve. Let's look at this with our tech debt time next week :)